10_code/threshold.py

```diff
@@ -114,23 +114,28 @@
 Texas, deaths from 2003 to 2014, threshold is 2007
 Control states = "TX","UT","GA","CO", "CA","ND","IL", "LA","MD","OK",
 """
-group_tx = ["TX","UT","GA","CO", "CA","ND","IL", "LA","MD","OK"]
-group_tx_names = ["Texas","Utah","Georgia","Colorado", "California","North Dakota","Illinois","Louisiana","Maryland","Oklahoma"]
+texas_states = ["TX","UT","GA","CO", "CA","ND","IL", "LA","MD","OK"]
```

Comment:
### Combine texas_states and texas_names
I think we should keep these in a dictionary of { state name: abbreviation  }. Refer to _data.py line 171._

Reply:
Yes I was initially using list but after the code started throwing errors I realised I had to map the population data which had full States names to the abbreviations  for merging with the overdose dataset, so i created a dictionary later.